### PR TITLE
Resolve failing tests by removing private

### PR DESCRIPTION
**Pull request by wally-yawn**
Quick refactor to fix broken tests

### CODE REVIEW

**Review by Crosswolfv1 — APPROVED** [review]
Approved, will address if private is needed and how to allow RSpec access

**Comment by Montana-Pfeifer** [comment]
viewed after merge and these are needed for function of tests

**Review by Montana-Pfeifer — COMMENTED** [review]
viewed after merge and these are needed for function of tests

**Pull request merged**
